### ISSUE TITLE
bulk upload: enable AC to perform bulk upload via the webapp

### ISSUE THREAD

**Issue by redshiftzero** [lucyparsons/OpenOversight#718]
Right now we enable administrators to, at the command line, upload officers in bulk via a CSV import: https://github.com/lucyparsons/OpenOversight/blob/develop/OpenOversight/app/commands.py#L341

We should enable admins and area coordinators of a given department (but only the department they are area coordinator of) to upload a CSV via the web application.

**Comment by am-i-the-rabbit** [comment]
This would actually help us tremendously right now. Has there been any more development from this idea since it was proposed?

**Comment by brianmwaters** [comment]
I think it's a good idea, and it would be easy enough to do by wrapping the bulk import commands in a web interface, but I would have big concerns about the attack surface (or just risk of benign instability) exposed by doing so, as the bulk import scripts are kind of ad-hoc and not really designed for this.

**Comment by abandoned-prototype** [comment]
I agree with @brianmwaters . it wouldn't be hard to feed any csv into the existing command, but I am also afraid of many unintended effects. Basically, to make this command useful it has to have so much access that it can also very easily break stuff.

Generally, I think it would be a great functionality, but would need a lot of additional work. In my opinion we would at least need a preview where people can get an overview on the chances they are about to make, as well as storing some kind of change log that allows us to also undo all the changes again.
